<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taiko</class>
 <widget class="QMainWindow" name="Taiko">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taiko</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BlueVolley">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>420</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Taiko-Desktop_Qt_5_6_0_MinGW_32bit-Debug/image/volleyball.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Z_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>424</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="X_text">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>425</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time_LCD">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Score_LCD">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Pika">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Taiko-Desktop_Qt_5_6_0_MinGW_32bit-Debug/image/025Pikachu_XY_anime_3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Volley">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Taiko-Desktop_Qt_5_6_0_MinGW_32bit-Debug/image/volleyball-ball-4235.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GameOver">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedVolley">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>414</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Taiko-Desktop_Qt_5_6_0_MinGW_32bit-Debug/image/QH6L2w0a_400x400.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="yourscore">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(162, 29, 230);</string>
    </property>
    <property name="text">
     <string>Your score:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="finalscore">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
